config/monitoring/metrics/opencensus/100-ocagent.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Based on https://github.com/census-instrumentation/opencensus-service/raw/master/example/k8s.yaml
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: opencensus-agent-config
+  namespace: knative-monitoring
```

Comment:
If this is intended to long-term replace the existing exporters, should we default deploy to knative-serving?

knative-monitoring has thus far been an optional add-on not required for metrics, but deploying some sort of oc-agent will be necessary in the future to see knative-serving metrics.

Reply:
I think the agent would be used for both serving and eventing (since they would both be picking up the changes in `pkg`). Furthermore, I think it would make more sense to have the agent be a single service across both (and possibly also providing a service to the applications on the cluster).